spec/bcrypt/password_spec.rb

```diff
@@ -26,6 +27,28 @@
     expect { BCrypt::Password.create( ""         ) }.not_to raise_error
     expect { BCrypt::Password.create( String.new ) }.not_to raise_error
   end
+
+  context 'with a 256 character password' do
+    before :each do
+      @secret = SecureRandom.hex(256)
```

Comment:
```suggestion
      @secret = SecureRandom.hex(128)
```

The character string returned from SecureRandom.hex is actually 2n characters long, per the docs:

> The argument n specifies the length, in bytes, of the random number to be generated. The length of the resulting hexadecimal string is twice of n.

Reply:
Aside from that, though, what's the value of these tests?  Just that they red/green the InvalidHash error on Ubuntu 20?  It feels like they should be more explicit if that's the goal.

Reply:
I wrote those before I wrote the specs in engine_spec and just left them. Removed them in favor of a more generic "long secret" test in the same style as the rest of the tests in the file. This would reveal the error on Ubuntu 20/Debian 11, but is generic enough to be sane on any platform. PTAL